docs-v2/pages/connected-accounts/oauth-clients.mdx

```diff
@@ -0,0 +1,56 @@
+- Pipedream apps solve for a broad range of use cases, which means the scopes requested by Pipedream's official OAuth client may request a different set of scopes than your specific use case.
+- To define the exact scope of access you'd like to grant, you can configure a custom OAuth client for most OAuth apps in Pipedream.
+
+## Configuring Custom OAuth Clients
+<ArcadeEmbed
+    src="https://demo.arcade.software/z8fudNnX8ScQbQI0wzIL?embed"
+    title="Create Custom OAuth Client in Pipedream">
+</ArcadeEmbed>
+
+<Steps>
+### Create an OAuth client in the relevant app
+For example, if you want to use a custom OAuth client for GitHub, you'll need to locate their documentation and create an OAuth app in your developer settings.
+
+### Navigate to the OAuth Clients page in Pipedream
+Open the [OAuth Clients page in your Pipedream account](https://pipedream.com/@/accounts/oauth-clients) and click "New OAuth Client".
+
+### Select the app
+Choose the app you need. If you can't find what you're looking for, feel free to [submit an integration request](https://pipedream.com/support).
+
+### Enter the required fields
+- **Name:** Give the OAuth client a name so it's easy to identify
+- **Description:** Optionally add a brief description for additional context
+- **Client ID and Secret**: Paste these values from the app's settings that you're configuring (the client secret is sensitive – we'll encrypt and hide it from the UI)
+- **Redirect URI:** Copy this Redirect URI and paste it into the app's settings
+- **Scopes:** We'll list the scopes from Pipedream's official OAuth client by default. Add or remove scopes as needed based on your use case.
+
+And finally, click "Save".
+
+<Callout type="warning">
+Make sure to include all the scopes you need based on your use case. You can modify the scopes later (you'll need to reconnect your account for changes to take effect). Refer to the app's API documentation for information on what scopes you'll need.
+</Callout>
+
+</Steps>
+
+## Connecting your account with with a custom OAuth client
+Once you've created the OAuth client, anyone in your workspace can connect their account:
+
+<br />
+
+<ArcadeEmbed
+    src="https://demo.arcade.software/hljzHunRakVX30QipJ1y?embed"
+    title="Connect using custom OAuth client">
+</ArcadeEmbed>
+
+Now you're ready to use the connected account in any workflow, just like any other account in Pipedream:
+
+![Connected accounts with OAuth client labels](https://res.cloudinary.com/pipedreamin/image/upload/v1717104725/oauth-clients-dropdown-labels_zzmycx.png)
+
+### Limitations
+- The vast majority of OAuth apps in Pipedream with with custom OAuth clients. However, due to the unique integration requirements for **Discord**, **Dropbox**, **Slack**, and **Zoom**, you can only use Pipedream's official OAuth client in **triggers** for those apps.
```

Comment:
I'd actually add a `#### Apps that don't allow custom clients` section here and then change the section to:

```markdown
The majority of OAuth apps in Pipedream support custom OAuth clients. However, due to the unique integration requirements for these apps, you can you can only use Pipedream's official OAuth client in **triggers** for those apps:

- Discord
- Dropbox
- Slack
- Zoom
```

Just to make the list of apps a little easier to scan